src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPages(from: number, to: number): number[] {
+  const numbers = [];
+
+  for (let n = from; n <= to; n += 1) {
```

Comment:
You also can use Array.from method to create an array with specified range. For example, const numbers = Array.from({length: to - from + 1}, (_, i) => from + i);